src/test/resources/out/compare/ietf-netconf-common-analyzed

```diff
@@ -41,3 +41,5 @@ units: 3
 uses: 1
 value: 3
 when: 1
+
```

Comment:
It should not expect 2 empty lines at the end of file.

Reply:
this commit was removed, since this issue will not be fixed.